backend/README.md

```diff
@@ -108,6 +108,59 @@ Later, once you have created an edition, replace `edition_name_here` with the na
 Replacing spaces in the name with `%20`. `My edition` becomes `My%20edition`
 This allows you to have multiple forms, each linked to a different edition.
 
+Because the ids of the questions from the tally form differ for each owner of the form, you will need to fill these in yourself.
+In `./src/main/resources/tally-ids.properties.example` you can find all of the question names that need to be mapped to their ids. 
+Just above each mapping you can find the full question. In this file all occurences of `question_id_here` will need to be replaced with the corresponding question id.
+You can get these ids by going to the webhook integration on your tally form and sending a test request. Then you should be able to see the following json (with different values):
+```json
+{
+  "eventId": "eb3e6aee-c64f-4e23-a39d-bbbd4018145a",
+  "eventType": "FORM_RESPONSE",
+  "createdAt": "2022-04-11T12:52:28.736Z",
+  "data": {
+    "responseId": "mR2o8P",
+    "submissionId": "mR2o8P",
+    "respondentId": "mexYPe",
+    "formId": "wMBOEn",
+    "formName": "(UGent) #osoc22 student application form",
+    "createdAt": "2022-04-11T12:52:28.000Z",
+    "fields": [
+      {
+        "key": "question_wAveXB",
+        "label": "Will you live in Belgium in July 2022?*",
+        "type": "MULTIPLE_CHOICE",
+        "value": "3095d1bf-41ba-42eb-a12c-d53b8ba242ea",
+        "options": [
+          {
+            "id": "3095d1bf-41ba-42eb-a12c-d53b8ba242ea",
+            "text": "Yes"
+          },
+          {
+            "id": "b456ab1d-3c73-4fc2-b56f-903690878239",
+            "text": "No"
+          }
+        ]
+      },
+      ...
+```
+
+The fields section is the important part here. This section contains all questions and their corresponding keys. 
```

Comment:
I did not have this locally, but I think it is because of the ... not being valid json syntax, see https://github.com/atom/language-gfm/issues/21#issuecomment-49513323.

Reply:
This should be fixed now.